packages/astro/src/core/render/result.ts

```diff
@@ -44,6 +44,7 @@ export interface CreateResultArgs {
 	status: number;
 	locals: App.Locals;
 	cookies?: AstroCookies;
+	currentLocale: string | undefined;
```

Comment:
I put `currentLocale: string | undefined;` because it helps me find the cases where `currentLocale` isn't passed. Like this, I was able to find all the spots where I needed to add the locale. While I agree with you, the solution I adopted is more sound than `currentLocale?: string`

Reply:
Cool! Just making sure it was an intentional decision. Carry on!